apps/web-giddh/src/app/new-inventory/component/inventory-sidebar/inventory-sidebar.component.ts

```diff
@@ -14,26 +15,40 @@ interface SidebarNode {
     icons?: string;
     name: string;
     link?: string;
+    moduleType?: string;
+    openActiveMenu?: boolean;
     children?: SidebarNode[];
 }
 const TREE_DATA: SidebarNode[] = [
```

Comment:
check and make this blank array since we are updating after translations loading